main_test.go

```diff
@@ -34,13 +34,13 @@ import (
 )
 
 func BenchmarkKubeStateMetrics(t *testing.B) {
```

Comment:
Nit: I would rename this to be `(b *testing.B)` as it's a benchmark and makes things more clear onwards.